plugin.video.mazacka/addon.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<addon id="plugin.video.mazacka"
+version="1.0"
+name="Mazačka - Streets of Prague"
+provider-name="Radek Homola">
+<requires>
+  <import addon="xbmc.python" version="2.25.0"/>
+</requires>
+<extension point="xbmc.python.pluginsource" library="main.py">
+  <provides>video</provides>
+</extension>
+<extension point="xbmc.addon.metadata">
+  <summary lang="en">Take a live tour through the streets of Prague, Czech Republic.</summary>
+  <summary lang="cs">Projeďte se speciální mazací tramvají ulicemi Prahy.</summary>
+  <description lang="en">Unique Czech tram called 'Mazačka' travels around the city of Prague. It does not stop to pick up passengers. Rather, ensures that all the other trams ride as smoothly as possible - by oiling the rails along its way. Tatra T3 tram number 5572 now has one more function: it provides an unexpected look at the 1000 year old city.</description>
+  <description lang="cs">Mazací tramvaj s evidenčním číslem 5572, pro kterou se vžil zkrácený název "Mazačka", slouží na pražských tramvajových tratích od ledna 2015. Aplikace maziva pomáhá snížit hluk a opotřebení kolejnic a kol. Každý den najede Mazačka v průměru přes 200 kilometrů.</description>
+  <platform>all</platform>
```

Comment:
maybe add:
`<language>cz</language>` if the content of the videos provided is czech only